test/SILOptimizer/OSLogFullOptTest.swift

```diff
@@ -235,7 +244,8 @@ func testDynamicPrecisionAndAlignment() {
     // First argument bytes.
     //
     // CHECK-NEXT: [[OFFSET2:%.+]] = getelementptr inbounds i8, i8* [[BUFFER]], i{{.*}} 2
-    // CHECK-NEXT: store i8 16, i8* [[OFFSET2]], align 1
+    // TODO: Figure out why "store i8 16" turned into "store i8 0"
```

Comment:
I think that this comment appears in the other places and is possibly a copy-paste error. If I remember correctly, there was a bug here that got fixed. I guess type `count` (represented by the value 16) is used only for precision but not for alignment. So the current patch has the right logic  and the earlier one was a bug. So I guess you can remove the `TODO` here and also in other places.